src/Stryker.Core/Stryker.Core/Initialisation/CsharpProjectComponentsBuilder.cs

```diff
@@ -105,7 +105,7 @@ private FolderComposite FindProjectFilesUsingBuildalyzer(ProjectAnalyzerResult a
                     continue;
                 }
 
-                var relativePath = Path.GetRelativePath(projectUnderTestDir, sourceFile);
+                var relativePath = Path.GetRelativePath(projectRoot, sourceFile);
```

Comment:
I don't think this is correct. the projectRoot is the 'solution folder'. Project files are not relative to the solution folder, the are relative to the project folder. Which is the projectUnderTestDir.

Reply:
I will take a look at our project structure to see if I can make some sense of it. It is perhaps too deep in stryker for a contributor to dig in to with the amount of available documentation (none 😅)

Reply:
I see things are a bit broken with some inconsistencies here and there because to my understanding, `RelativePath` means relative to root folder and `RelativePathToProjectFile` means relative to project folder, so they can't be the same unless the project is on the root folder.

For example, in line 116, this is how `RelativePath` is set: `RelativePath = _fileSystem.Path.Combine(folderComposite.RelativePath, fileName),` which leads to a wrong path.

Maybe I don't understand exactly what we're trying to achieve here.

Reply:
A contributor added the RelativePathToProjectFile at some point and we failed to question why he did that. At this point I'm not entirely sure what we're trying to achieve here either so I'm gonna have to look it over with a colleague and decide on how we want the design to be. Then fix up the whole thing and document what's what and why so we don't mess it up again. It might take us some time to get to that. Apologies.